### PR TITLE
new types: readable-to-readable

### PR DESCRIPTION
**Pull request by tpluscode**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/readable-to-readable/index.d.ts
1. types/readable-to-readable/readable-to-readable-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 42429ed038de2f0d9f1727c221b2d73723e45915
</p>

File: types/readable-to-readable/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "devDependencies": {
+        "@types/readable-to-readable": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Tomasz Pluskiewicz",
+            "githubUsername": "tpluscode"
+        },
+        {
+            "name": "ludovicm67",
```
Comment (ludovicm67) [inline types/readable-to-readable/package.json:20]:
```suggestion
            "name": "Ludovic Muller",
```

**Review by ludovicm67 — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
@tpluscode thanks!

**Comment by tpluscode** [comment]
Ready to merge

Sent from Nine
________________________________

**Pull request merged**